### ISSUE TITLE
output search result to file selection dialog?

### ISSUE THREAD

**Issue by chchia** [Flow-Launcher/Flow.Launcher#1117]
![image](https://user-images.githubusercontent.com/1291771/162106054-1d35a577-353f-43b0-9a62-e275aa6ce484.png)

for example, Listary have this fantastic feature to output the selected search result to file selection dialog, is Flow.Launcher having something same or similar?

i tried to search for it but did not found one.

**Comment by taooceros** [comment]
Not for now. We do want to implement it, but there's quite a bit technical difficulty.

**Comment by vagra** [comment]
I really like this feature, it's such a pain to find some deep paths in the open/save file dialog now, lastary has been abandoned by the author, and everything doesn't have this feature.

**Comment by rockingmax** [comment]
Please add this feature. It's really useful!

**Comment by aashish108** [comment]
Yeah, Listary and Fluent Search have this. Its so useful. Subbing.
